packages/manager/apps/pci-instances/src/hooks/instance/useInstancesTest.ts

```diff
@@ -0,0 +1,27 @@
+import { useProjectUrl } from '@ovh-ux/manager-react-components';
+import { useProjectId } from '../project/useProjectId';
+import {
+  TUseInstancesQueryParams,
+  useInstances,
+} from '@/data/hooks/instance/useInstances';
+
+export const useInstancesTest = ({
+  limit,
+  sort,
+  sortOrder,
+  filters,
+}: TUseInstancesQueryParams) => {
+  const projectUrl = useProjectUrl('public-cloud');
+  const projectId = useProjectId();
+
+  if (!projectId) {
+    throw new Error('invalid projectID');
+  }
+
+  return useInstances(projectId, projectUrl, {
```

Comment:
```suggestion
  return useInstances(projectId, { projectUrl }, {
```
Maybe change to this to make it more generic